libs/git_wrapper/Git_wrapper.mli

```diff
@@ -25,6 +25,10 @@ val files_from_git_ls : cwd:Fpath.t -> Fpath.t list
 (* get merge base between arg and HEAD *)
 val get_merge_base : string -> string
 
+(* get repository root path *)
+val get_git_root_path : unit -> string
+val chdir_to_repo_root : unit -> unit
```

Comment:
define instead a find_git_root_path_in_parents that returns an Fpath.t
and say in the comment that this may raise invalid_arg.